Cargo.toml

```diff
@@ -34,6 +34,8 @@ atmega644 = []
 attiny84 = []
 attiny85 = []
 attiny88 = []
+attiny841 = []
+attiny861 = []
```

Comment:
I know it's a bit weird but please adhere to the alphabetical sorting that is present so far:

```diff
 attiny84 = []
+attiny841 = []
 attiny85 = []
+attiny861 = []
 attiny88 = []
```

Same goes for all the other changes as well.

Reply:
Oops, I totally intended to do that, but for some reason my brain said that 41 is bigger than 8. My bad